src/t8_schemes/t8_standalone/t8_standalone_elements.hxx

```diff
@@ -0,0 +1,141 @@
+//                                                            [T8_ELEMENT_NUM_FACES[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_type_face_to_is_1_boundary[1 << T8_ELEMENT_NUM_EQUATIONS[TEclass]]
+//                                                        [T8_ELEMENT_NUM_FACES[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_type_face_to_last_facechilds_cubeid[1 << T8_ELEMENT_NUM_EQUATIONS[TEclass]]
+//                                                                 [T8_ELEMENT_NUM_FACES[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_type_face_facechildid_to_childid[1 << T8_ELEMENT_NUM_EQUATIONS[TEclass]]
+//                                                              [T8_ELEMENT_NUM_FACES[TEclass]]
+//                                                              [T8_ELEMENT_MAX_NUM_FACECHILDREN[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_type_childid_face_to_childface[1 << T8_ELEMENT_NUM_EQUATIONS[TEclass]]
+//                                                            [T8_ELEMENT_NUM_CHILDREN[TEclass]]
+//                                                            [T8_ELEMENT_MAX_NUM_FACECHILDREN[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_type_face_to_tree_face[1 << T8_ELEMENT_NUM_EQUATIONS[TEclass]]
+//                                                    [T8_ELEMENT_NUM_FACES[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_rootface_dim_to_facedim[T8_ELEMENT_NUM_FACES[TEclass]][T8_ELEMENT_DIM[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_rootface_eq_to_faceeq[T8_ELEMENT_NUM_FACES[TEclass]][T8_ELEMENT_NUM_EQUATIONS[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr int8_t t8_sele_lut_type_rootface_to_face[1 << T8_ELEMENT_NUM_EQUATIONS[TEclass]]
+//                                                   [T8_ELEMENT_NUM_FACES[TEclass]];
+// template <t8_eclass_t TEclass>
+// constexpr t8_eclass_t t8_sele_lut_rootface_to_eclass[T8_ELEMENT_NUM_FACES[TEclass]];
+
+// #include "t8_sele_lut_triangle_cxx.hxx"
+// #include "t8_sele_lut_prism_cxx.hxx"
+// #include "t8_sele_lut_pyra_cxx.hxx"
+// #include "t8_sele_lut_tet_cxx.hxx"
+
+typedef uint32_t t8_element_coord;
+typedef uint8_t t8_element_level;
+typedef int8_t t8_cube_id;
+
+template <t8_eclass_t TEclass>
+using t8_element_type = std::bitset<T8_ELEMENT_NUM_EQUATIONS[TEclass]>;
+
+template <t8_eclass_t TEclass>
+using t8_element_coords = std::array<t8_element_coord, T8_ELEMENT_DIM[TEclass]>;
+
+template <t8_eclass_t TEclass>
+struct t8_standalone_element
+{
+  /** The refinement level of the element relative to the root at level 0. */
+  t8_element_level level;
+
+  /** The coordinates of the anchor vertex of the element. */
+  t8_element_coords<TEclass> coords;
+
+  /** Bit array: which inequality is fulfilled at which level. */
+  t8_element_type<TEclass> type;
```

Comment:
Is the type currently used anywhere? Except for setting it to zero in root and printing it and helper functions?
I would remove it for this PR, as the type is only used for non hypercube elements